packages/website/next.config.js

```diff
@@ -18,11 +18,21 @@ module.exports = withCSS(
       // eslint-disable-next-line no-param-reassign
       config.externals = [];
 
+      config.module.rules.push({
+        test: /\.(ts|tsx)$/,
```

Comment:
What's the reason this ended up needing to be added in this PR? I would have expected this to be already be supported?

Reply:
it just refused to import a tsx component into the js file for some reason

Reply:
We need to tell webpack to use the babel loader for tsx files. Without this the babel loader is only used on .js files and .tsx ones are left untransformed.